docs/reference/index-modules/translog.asciidoc

```diff
@@ -131,6 +131,7 @@ Creating new empty checkpoint at [data/nodes/0/indices/P45vf_YQRhqjfwLMUvSqDw/0/
 Creating new empty translog at [data/nodes/0/indices/P45vf_YQRhqjfwLMUvSqDw/0/translog/translog-3.tlog]
 Done.
 --------------------------------------------------
+// NOTCONSOLE
```

Comment:
Here I'd change the snippet from `[source,js]` to `[source,txt]` which will cause it to skip the detection so you don't need `// NOTCONSOLE`.

Reply:
Good catch, I don't remember why I added this NOTCONSOLE